test/bdd/steps/network.js

```diff
@@ -513,10 +533,20 @@ Given(/^I start additional node[s]*$/, { timeout: 60000 }, function () {
     return Promise.all(additionalNodesStarts);
 });
 
-Given(/^DV publishes query consisting of path: "(\S+)", value: "(\S+)" and opcode: "(\S+)" to the network/, { timeout: 90000 }, async function (path, value, opcode) {
-    expect(!!this.state.dv, 'DV node not defined. Use other step to define it.').to.be.equal(true);
+Given(/^([DV|DV2]+) publishes query consisting of path: "(\S+)", value: "(\S+)" and opcode: "(\S+)" to the network$/, { timeout: 90000 }, async function (whichDV, path, value, opcode) {
```

Comment:
No need for all of this logic down. Just take desired DV from state:
```
const { dv } = this.state[whichDv];
```
Everything else is the same.

Reply:
you are abs right, will fix